models/events/webhook/github/Installation.js

```diff
@@ -0,0 +1,50 @@
+const Model = require('~/models/Model')
```

Comment:
I put everything in one file here because I haven't had a need to reference them individually yet. We can pull out later, but I thought it would look better for each file in models/events/webhook/github/ to correspond to an event. Having partials makes it harder to discern which are events just by looking at the directory structure.